src/Pages/Login/Login.js

```diff
@@ -1,25 +1,65 @@
-import React, { Component } from 'react';
+import React, { Component, useState } from 'react';
+import { Link } from 'react-router-dom';
 import styled from 'styled-components';
 
-const Login = () => {
+const Login = ({ history }) => {
+  const { Kakao } = window;
+
+  const KakaoLogin = () => {
+    Kakao.Auth.login({
+      success: function (authObj) {
+        console.log(authObj);
+        fetch('http://3.34.199.216:8000/users/kakaosignin', {
+          method: 'POST',
+          body: JSON.stringify({
+            access_token: authObj.access_token,
+            refresh_token: authObj.refresh_token,
+          }),
+        })
+          .then(res => res.json())
+          .then(res => {
+            console.log(res);
+            sessionStorage.setItem('Watta_token', res.access_token);
+            if (res.message === 'success') {
+              alert(`${res.user}님 Wa:tta Taxi에 오신것을 환영합니다!`);
+              history.push('/');
+            }
+            if (res.status === 401) {
+              alert('다시로그인 하세요');
+            }
+            if (res.status === 400) {
+              alert('이미 email로 가입된 기록이 있습니다.');
+            }
+          });
+      },
+      fail: function (err) {
+        alert('무언가 잘못되었습니다.');
+      },
+    });
+  };
+
+  const [isSignUp, setIsSignUp] = useState(false);
+
   return (
     <LoginContainer>
       <MovingIcon width="430" scrollamount="600">
         🚤
       </MovingIcon>
-      <Title>Welcome!</Title>
+      <Title>{isSignUp === true ? 'Join Wa:tta!' : 'Welcome!'}</Title>
```

Comment:
```suggestion
      <Title>{isSignUp ? 'Join Wa:tta!' : 'Welcome!'}</Title>
```
isSignUp이 이미 boolean 값을 가지고 있을 것이기 때문에 true와 비교하지 않아도 괜찮아보입니다